MAUI/Barcode/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: About .NET MAUI Barcode Generator control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Barcodes(SfBarcodeGenerator) control with key features and more.
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# .NET MAUI Barcode Generator Overview
+
+The Syncfusion .NET MAUI Radial Barcode Generator is a data visualization control used to generate and display data in a machine-readable format. It provides a perfect approach to encode text using supported symbology types.
+
+![.NET MAUI Barcode Genrator Control](images/overview/maui-one-dimensional-barcode.png)
+
+![.NET MAUI Barcode Genrator Control](images/overview/maui-two-dimensional-barcode.png)
+
+## Key features
+
+* **One-dimensional barcodes** - Barcode Generator supports different one-dimensional barcode symbologies such as `Code128`, `EAN8`, `EAN13`, `UPC-A`, `UPC-E`, `Code39`, `Code39 Extended`, `Code93`, and `Codabar`.
+* **Two-dimensional barcode** - Barcode Generator supports popular `QR code` and `Data Matrix`.
+* **Barcode customization** - Customize the visual appearance of barcodes using the backgroundColor and barColor properties and adjust the size of smallest line or dot of the code using the `module` property.
```

Comment:
Customize the visual appearance of barcodes using the `backgroundColor` and `barColor` properties, and adjust the size of the smallest line or dot of the code using the `module` property.

Reply:
Changes updated.